docs/docs/installation/configuration/namespace-keptn.md

```diff
@@ -0,0 +1,76 @@
+---
+comments: true
+---
+
+# Namespaces and Keptn
+
+This page discusses how to restrict the namespaces
+where Keptn and the
```

Comment:
why do we distinguish between Keptn and one of its components? I think this may be a bit confusing maybe we could keep Keptn all list all three components? Dunno

Reply:
Good point!  This needs a rewrite.  Let's be sure that I understand exactly what is being restricted.  Is it only restricting Keptn from "interrupting" deployments on the namespaces that are not specified?  So only lifecycle-operator and maybe the scheduler?

If I implement namespace restriction, is the metrics-operator prevented from monitoring namespaces that are not specified as allowed?  Will deployments that are not allowed be reflected in metrics and trace data?

Reply:
mm I see where you come from, in this case maybe I would refer to the feature more than the component, what about: "this page discuss how to restrict the lifecycle management feature of Keptn to only certain namespaces"

Reply:
Thanks!  This was just what I needed.  I used slightly different phraseology but see what you think.  If you approve, you can resolve this conversation.